docs/configuration.md

```diff
@@ -1,401 +1,406 @@
-Configuration
-=============
-
-
-Master Configuration
---------------------
-
-### master.database_directory
```

Comment:
Please don't change the line endings for files. Go into your editor and set the EOL or end of line convention of this file to Unix/LF and save it.

Reply:
yeah sorry